### PR TITLE
Check that cni subvolume was created

### PR DESCRIPTION
**Pull request by kravciak**

### CODE REVIEW

File: tests/caasp/one_line_checks.pm [hunk 1]
```diff
@@ -24,6 +24,9 @@ sub run {
     # bsc#1019652 - Check that snapper is configured
     assert_script_run "snapper list";
 
+    # https://build.opensuse.org/request/show/583954
```
Comment (sysrich) [inline tests/caasp/one_line_checks.pm]: Will not work - /var/lib/cni is NOT a subvolume on Kubic/CaaSP 4 and never will be (/var is a subvolume already)

**Comment by kravciak** [comment]
Fixed, thanks

File: tests/caasp/one_line_checks.pm [hunk 2]
```diff
@@ -24,6 +24,16 @@ sub run {
     # bsc#1019652 - Check that snapper is configured
     assert_script_run "snapper list";
 
+    # Subvolume check
+    if (is_caasp '4.0+') {
+        assert_script_run "btrfs subvolume show /var";
```
Comment (asmorodskyi) [inline tests/caasp/one_line_checks.pm]: it's matter of taste but I would say that more readable version would be 

```
my $path= is_caasp('4.0+') ? "/var" : "/var/lib/cni" ;
assert_script_run "btrfs subvolume show $path";
```

**Comment by asmorodskyi** [comment]
LGTM

**Pull request merged**